fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FetchResourcePayloadsDAO.java

```diff
@@ -181,14 +181,12 @@ public int count(Connection c) throws FHIRPersistenceException {
         final String select = query.toString();
 
         if (logger.isLoggable(Level.FINE)) {
-            Timestamp from = Timestamp.from(fromLastUpdated);
```

Comment:
This line caused a NPE when logging is turned on. 
We tested it with logging on/logging off.